### PR TITLE
operator cluster-aas-operator (0.1.5)

### PR DESCRIPTION
**Pull request by amisstea**
Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [ ] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [ ] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [ ] Have you tested your Operator with all Custom Resource Definitions?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [ ] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [ ] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Updates to existing Operators

* [ ] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [ ] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [ ] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [ ] Have you tested an update to your Operator when deployed via OLM?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [ ] Modify more than one operator
* [ ] Modify an Operator you don't own
* [ ] Rename an operator - please remove and add with a different name instead
* [ ] Modify any files outside the above mentioned folders
* [ ] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [ ] Description about the managed Application and where to find more information
2. [ ] Features and capabilities of your Operator and how to use it
3. [ ] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [ ] Human readable name and 1-liner description about your Operator
* [ ] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [ ] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [ ] Links to the maintainer, source code and documentation
* [ ] Example templates for all Custom Resource Definitions intended to be used
* [ ] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Comment by amisstea** [comment]
Attn @rawagner

**Comment by amisstea** [comment]
@rawagner do you have any clues as to what might be causing the CI failures? I see the controller manager pod is stuck on creation but otherwise I can't tell what the root cause is.

```
NAME                                                              READY   STATUS              RESTARTS   AGE
032a87f20f524f2ae801033aec7b419a1e79e44f8a5d7dceb818fc99ffgvmsk   0/1     Completed           0          14m
4858040cc62e08946163366ac7a2b19fd92030ef61522d70256b218b0fdjdsl   0/1     Completed           0          14m
argocd-operator-controller-manager-b6898f485-vxl4g                1/1     Running             0          14m
cluster-aas-operator-controller-manager-6fd44c4849-5vdf8          0/2     ContainerCreating   0          14m
test-operators-ocs-9q24j                                          1/1     Running             0          15m
test-operators-ocs-prod-9zl4w                                     1/1     Running             0          15m
```

I can pull the image I used (`quay.io/stolostron/cluster-templates-operator:2.11.0-0c3131154065397d37e3b66ae635d7ea88650bf9`) without issue.

**Comment by rawagner** [comment]
@amisstea Im not sure. Can you force-push to trigger the tests again ?

**Review by rawagner — CHANGES_REQUESTED** [review]

File: operators/cluster-aas-operator/0.1.5/manifests/cluster-aas-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,654 @@
...
+                    cpu: 5m
+                    memory: 64Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+              - args:
+                - --health-probe-bind-address=:8081
+                - --metrics-bind-address=127.0.0.1:8080
+                - --leader-elect
+                - --tls-cert-file=/etc/certs/tls/tls.crt
+                - --tls-private-key-file=/etc/certs/tls/tls.key
```
Comment (rawagner) [inline operators/cluster-aas-operator/0.1.5/manifests/cluster-aas-operator.clusterserviceversion.yaml:472]: this needs to be removed too

File: operators/cluster-aas-operator/0.1.5/manifests/cluster-aas-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,654 @@
...
+                  name: tls-cert
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: cluster-aas-operator-controller-manager
+              terminationGracePeriodSeconds: 10
+              volumes:
+              - name: tls-cert
+                secret:
+                  defaultMode: 420
+                  secretName: cluster-aas-operator-certs
```
[the remark this answers is not in the document]
Comment (amisstea) [reply operators/cluster-aas-operator/0.1.5/manifests/cluster-aas-operator.clusterserviceversion.yaml]: That did the trick. Thanks!

**Comment by amisstea** [comment]
@rawagner please merge this when you get a chance.

**Review by rawagner — APPROVED** [review]

**Pull request merged**